deps/chakrashim/src/inspector/v8-inspector-session-impl.cc

```diff
@@ -28,7 +31,9 @@ bool V8InspectorSession::canDispatchMethod(const StringView& method) {
          stringViewStartsWith(method,
                               protocol::Console::Metainfo::commandPrefix) ||
          stringViewStartsWith(method,
-                              protocol::Schema::Metainfo::commandPrefix);
+                              protocol::Schema::Metainfo::commandPrefix) ||
+         stringViewStartsWith(method,
```

Comment:
Since we already have code under ENABLE_TTD_NODE might be good to have all this under same

---
In reply to: [116629150](https://github.com/nodejs/node-chakracore/pull/244#discussion_r116629150) [](ancestors = 116629150)